collect_app/src/main/java/org/odk/collect/android/audio/AudioControllerView.java

```diff
@@ -101,10 +103,6 @@ private void onPositionChanged(Integer newPosition) {
         }
     }
 
-    private static String getTime(long seconds) {
-        return new DateTime(seconds, DateTimeZone.UTC).toString("mm:ss");
```

Comment:
What happens if you just do HH:mm:ss here? Does it fail beyond 24 hours? Does that matter?

Reply:
Yeah beyond 24 hours we roll back over. It's probably not a huge problem as I don't imagine people are recording/choosing audio that long but using `DateTime` is a little weird.

Reply:
Oh, I see. It always puts the hours which is not what we want.